multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -398,7 +371,7 @@ def __init__(
             set_logger_verbosity(verbosity, logger=logger)
 
         self._label_column = label
-        self._problem_type = problem_type if problem_type is not None else None
+        self._problem_type = problem_type
```

Comment:
Should be `problem_type.lower() if problem_type is not None else None`?

Reply:
It has been called previously.